protocols/direct-fund/direct-fund.go

```diff
@@ -254,6 +270,37 @@ func (s DirectFundObjective) clone() DirectFundObjective {
 
 	return clone
 }
+func (s DirectFundObjective) createSignedStateMessages(ss state.SignedState) []protocols.Message {
+
+	messages := make([]protocols.Message, 0)
+	for i, participant := range ss.State().Participants {
+
+		// Do not generate a message for ourselves
+		if uint(i) == s.C.MyIndex {
+			continue
+		}
+		message := protocols.Message{To: participant, ObjectiveId: s.Id(), SignedStates: []state.SignedState{ss}, Proposal: nil}
+		messages = append(messages, message)
+	}
+	return messages
+}
+
+func (s DirectFundObjective) signAndStore(toSign state.State, secretKey *[]byte) (state.SignedState, error) {
+
+	sig, err := toSign.Sign(*secretKey)
+	if err != nil {
+		return state.SignedState{}, fmt.Errorf("failed to sign state: %w", err)
+	}
+	ss := state.NewSignedState(toSign)
+	if err := ss.AddSignature(sig); err != nil {
+		return state.SignedState{}, fmt.Errorf("failed to add signature to signed state: %w", err)
+	}
+	ok := s.C.AddSignedState(ss)
```

Comment:
I wonder if we could save some lines by using 

https://github.com/statechannels/go-nitro/blob/3c4925d201fe18bf5fe768b107565e8b3f4a7c47/channel/channel.go#L170-L171